zulip_bots/zulip_bots/bots/github_detail/github_detail.py

```diff
@@ -74,8 +74,8 @@ def handle_message(self, message: Dict[str, str], bot_handler: Any) -> None:
             return
 
         # Capture owner, repo, id
-        issue_prs = re.finditer(
-            self.HANDLE_MESSAGE_REGEX, message['content'])
+        issue_prs = itertools.islice(re.finditer(
```

Comment:
I think `itertools.islice` is overkill for this.  I'd also break it into two lines:

~~~
issue_prs = re.finditer(....)

# We limit to 5 requests to prevent denial-of-service
issuer_prs = issue_prs[:5]
~~~

Technically, we should return an error message is somebody tries to submit 5+ requests.

Reply:
fuck